src/renderer/components/RecentConnections.tsx

```diff
@@ -13,7 +13,15 @@ const RecentConnections = () => {
     };
     getConnections();
   }, []);
-
+  const handledelete = async (ConnectionID: number) => {
+    const index = connect.findIndex(
+      (connection) => connection.id === ConnectionID
+    );
+    window.connections.ipcRenderer.delete(index);
```

Comment:
```suggestion
    window.connections.ipcRenderer.delete(ConnectionID);
```

Reply:
Also this is unrelated to this PR, but @adityavidyadharan and @JackFratto , do we need to add some logic to where if we delete the current connection, we automatically disconnect from it, or is that handled already

Reply:
That's not necessary since you get to this menu while in the disconnected state. You only connect after clicking something on the menu, and to get back here you would first disconnect